server/routes/newSeats.js

```diff
@@ -0,0 +1,76 @@
+// Get dependencies
+const express = require('express');
+const router = express.Router();
+const mongodb = require('mongodb');
+const post = require()
+
+router.get('/', (req, res) => {
+res.send([]);
```

Comment:
Ideally we want to define schemas (models) in a sperate file and then require them into this file.

See layout of 
https://github.com/gothinkster/node-express-realworld-example-app